pkg/apis/networking/types.go

```diff
@@ -26,6 +26,8 @@ type AppliedToGroup struct {
 	metav1.ObjectMeta
 	// Pods is a list of Pods selected by this group.
 	Pods []GroupMemberPod
+	// GroupMembers is a list of resources selected by this group.
+	GroupMembers []GroupMember
```

Comment:
So if I understand this correctly the externalEntities should be computed and stored in GroupMember, while current code usage of GroupMemberPod will be migrated to GroupMember later?

Reply:
yes, that is the intention